engine/rig/proto/rig/rig_ddf.proto

```diff
@@ -166,62 +101,58 @@ message MeshVertexIndices
     optional uint32 position    = 1;
     optional uint32 texcoord0   = 2;
     optional uint32 normal      = 3;
+    // TODO: Add tangents and texcoord1 here as well
```

Comment:
We still keep all these indices, in order for us to recreate the buffers of different lengths, depending on the GPU hardware support (e..g 32 bit indices or not).

Personally, I'd like to remove that support, since I don't think it's really used, and if our users should create content that is appropriate for their targets GPU.